src/node/rpc/member_frontend.h

```diff
@@ -554,41 +554,41 @@ namespace ccf
     //! Table of functions that proposal scripts can propose to invoke
     const std::unordered_map<
       std::string,
-      std::function<bool(ObjectId, kv::Tx&, const nlohmann::json&)>>
+      std::function<bool(ProposalId, kv::Tx&, const nlohmann::json&)>>
```

Comment:
Minor: Now that these are `std::string` rather than `size_t`, we should (where possible) pass them by `const&`.

Reply:
Good point, will fix.